app/service/operation_svc.py

```diff
@@ -84,14 +84,14 @@ def __init__(self):
                                         attack=dict(tactic=ability['tactic'],
                                                     technique_name=ability['technique_name'],
                                                     technique_id=ability['technique_id'])))
-        return report
+        return json.dumps(report, indent=4)
 
     """ PRIVATE """
 
     @staticmethod
     async def _write_report(report):
         with open(os.path.join('logs', 'operation_report_' + report['name'] + '.json'), 'w') as f:
-            f.write(json.dumps(report, indent=4))
```

Comment:
hmm.. I'm not sure on this one..  if you're getting the operation report, I think we'd want to keep that programmatic and not make styling choices (indent). I do like the JSON conversion though... thinking...

Reply:
I'm not sure on this one either

Reply:
closed

Reply:
see https://github.com/mitre/chain/pull/135